mods/ra/chrome/editor.yaml

```diff
@@ -172,6 +173,16 @@ Background@SAVE_MAP_PANEL:
 			Font: Bold
 			Key: escape
 
+ScrollPanel@MAP_SAVE_VISIBILITY_PANEL:
+	Width: 220
+	Height: 44
```

Comment:
Might want to increase this to avoid the up and down buttons overlapping with the scroll bar.